POKEMON_OF_THE_DAY/PANDIUM.yaml

```diff
@@ -5,37 +5,12 @@ run: node .
 configs:
   schema:
     properties:
-      pokemon_type:
-        type: string
-        $ref: '#/definitions/pokemon_types'
-      slack_user:
-        type: string
-        $ref: '#/definitions/slack_users'
-    required:
-      - pokemon_type
-      - slack_user
-    definitions:
-      slack_users:
-          type: number
-          oneOf:
-              -   title: Placeholder
-                  const: placeholder
-      pokemon_types:
-          enum:
-              -  placeholder  
+      placeholder:
+          type: string
```

Comment:
you are correct that we need the placeholder there to have an integration with no configs.

Reply:
I can make a ticket for it but we've never had an integration with no configs. And it will be very unlikely we ever do in real use case

Reply:
Whether or not there is a use case for a finished integration with no configs, it's certainly a situation that can arise. Users have complete control over their pandium.yaml files. 

This is a bug; looking at the relevant page in the admin dash we do try to handle empty configs, but it must not be working.  We need to fix bugs when we  find them, rather than working around them.... particularly in such as public matter, putting convoluted placeholders in a tutorial

Reply:
Alrighty- [this pull request ](https://github.com/pandium/theproduct/pull/2019 ) alters source control so that we can build releases for integrations that don't have `configs` in their PANDIUM.yaml
I have therefore removed the configs from the PANDIUM.yaml all together form this sample.